server/schedule/operator/step.go

```diff
@@ -682,3 +739,12 @@ func validateStore(ci ClusterInformer, id uint64) error {
 	}
 	return nil
 }
+
+func maxWaitTime(executorRate float64, regionSize int64) time.Duration {
```

Comment:
Can we just change it to `func maxWaitTime(regionSize int64) time.Duration` and use the executorRate as a constant internally so that we don't need to set it or add an extra field in the operator?